### ISSUE TITLE
About codes

### ISSUE THREAD

**Issue by guoyanb** [vamships/PECAN#2]
Hello! I am very interested in your work. When I performed experiments,  I could not find the inplementation of your codes about "For each combination, networks were trained until the performance on the validation set stopped improving or for a maximum of 250 epochs". Could you kindly release the codes?  Thanks!

**Comment by vamships** [comment]
Hi,
Thanks for your interest in PECAN! The function `fit_model_and_activate` in the script `train_test.py` handles the training part. There's a `train_test.py` script in each of the three GCN_x folders, so make sure to use the one corresponding to your task.

**Comment by guoyanb** [comment]
> Hi,
> Thanks for your interest in PECAN! The function `fit_model_and_activate` in the script `train_test.py` handles the training part. There's a `train_test.py` script in each of the three GCN_x folders, so make sure to use the one corresponding to your task.

Thanks for you reply ! For the text "For each combination, networks were trained until the performance on the validation set stopped improving or for a maximum of 250 epochs", does it mean that you did all the experiments for a maximum of 250 epochs ? Maybe, did you use the early-stopping strategy?

**Comment by vamships** [comment]
Yes, I trained the models for a maximum of 250 epochs, or until validation set performance stopped improving. The test set performance was reported on whichever epoch the model was trained up to.

**Comment by guoyanb** [comment]
Thanks!

**Comment by guoyanb** [comment]
> Yes, I trained the models for a maximum of 250 epochs, or until validation set performance stopped improving. The test set performance was reported on whichever epoch the model was trained up to.

Thanks!
